packages/map/src/components/UsaMap/components/UsaMap.County.tsx

```diff
@@ -11,7 +11,16 @@ import ErrorBoundary from '@cdc/core/components/ErrorBoundary'
 
```

Comment:
did you mean for changes to this file to be in your PR?

Reply:
I don't remember making these changes or why I would've done it. I know that you made some changes before I took over the branch. If this isn't part of  your changes then no. I will remove them.